docker/gpu-cuda/jax.dockerfile

```diff
@@ -0,0 +1,6 @@
+From pennylane/cuda/base:latest
+
+# Update and install Jax
+RUN apt-get update && pip3 install jax jaxlib==0.1.67+cuda111 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```

Comment:
Tests must be rerun.

Also:
```
jaxlib==0.1.67+cuda111
```
I think the above package version should be specified somewhere that we have ease-of-access to update. Is there a way to ensure we pull in a jaxlib with CUDA support that matches the jax version, without explicitly listing a version number? If not, we should also pin jax to the same version.

Reply:
Sure, I will verify that part. as much I understand JAX has different versions fro CPU and  GPU or may be it's true for JAXLIB only. Will investigate this one.